### PR TITLE
Full card game with rounds and end state and remote decks

### PR DESCRIPTION
**Pull request by BlandSchwad**
The axios request for the remote decks takes place in CreateGame.js. It's currently hardcooded to an EC2 server IP, but can be easily changed for local development. 

Game is now fully playable from start to finish with viewable cards and an end state that does a simple calculation to find the winner.

### CODE REVIEW

File: src/features/CreateGame.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import React, { useReducer } from "react";
 import { useNavigate } from "react-router-dom";
+<<<<<<< HEAD
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: Resolve these

File: src/features/CreateGame.js [hunk 2]
```diff
@@ -20,6 +23,7 @@ import {
 } from "../styles/createGameStyles";
 import { Box, Typography } from "@mui/material";
 // import { StyledComponentContainer } from "../styles/globalStyles";
+>>>>>>> main
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: Resolve these

File: src/features/CreateGame.js [hunk 3]
```diff
@@ -30,7 +34,41 @@ export function CreateGame() {
   );
   const { name, options, customCards } = CreateGameState;
 
+<<<<<<< HEAD
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: Resolve these

File: src/features/CreateGame.js [hunk 4]
```diff
@@ -30,7 +34,41 @@ export function CreateGame() {
...
+    let result = await axios.get('http://3.101.13.217:45000/cards/answer')   
+    setOptions(previousOptions => {  return {...previousOptions, setupData: {...previousOptions.setupData, remoteAnswerDeck: result.data}}  })
+  
+
+    
+  };
+  console.log(options);
+
+  const clickHandler = () => {
+=======
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: Resolve these

File: src/features/CreateGame.js [hunk 5]
```diff
@@ -30,7 +34,41 @@ export function CreateGame() {
...
   const createGameHandler = () => {
+>>>>>>> main
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: Resolve these

File: src/features/CreateGame.js [hunk 6]
```diff
@@ -67,6 +105,22 @@ export function CreateGame() {
   };
 
   return (
+<<<<<<< HEAD
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: resolve these

File: src/card/Card.js [hunk 7]
```diff
@@ -10,7 +11,7 @@ import {
 export default class Card extends React.Component {
   render() {
     const shareTitle = 'Apples to Oranges!';
-    const shareMessage = `I'm playing Apples to Oranges and found this hilarious!\n${this.props.children}`;
+    const shareMessage = `I'm playing Apples to Oranges and found this hilarious!\n${this.props.text}`;
```
Comment (codingavatar) [inline src/card/Card.js:27]: I think this will need to be {this.props.children} instead of {this.props.text}

File: src/features/CreateGame.js [hunk 8]
```diff
@@ -67,6 +105,22 @@ export function CreateGame() {
...
+        flexDirection: "column",
+        justifyContent: "space-between",
+        ml: { sm: "0rem", md: "4rem" },
+        mt: "2rem",
+      }}
+    >
+      <Typography variant="h4">Create a Game</Typography>
+      <FormGroup
+        sx={{ height: "18rem", justifyContent: "space-evenly", mt: "1rem" }}
+=======
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: Needs to resolve these

File: src/features/CreateGame.js [hunk 9]
```diff
@@ -83,6 +137,7 @@ export function CreateGame() {
           color: "white",
         }}
         variant="body2"
+>>>>>>> main
```
Comment (crypto-bender) [inline src/features/CreateGame.js]: Resolve these

**Review by crypto-bender — CHANGES_REQUESTED** [review]
Resolve conflicts/clean merge code and its good

**Review by crypto-bender — APPROVED** [review]

**Review by codingavatar — APPROVED** [review]
Great!

**Pull request merged**